B827EBFFFE7DB2A6.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE7DB2A6", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-us-west", 

```

Comment:
I think this is not the correct server address. Check the options here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses